pkg/virt-config/config-map.go

```diff
@@ -59,6 +59,8 @@ const (
 	MigrationCompletionTimeoutPerGiB         int64  = 800
 	DefaultMachineType                              = "q35"
 	DefaultCPURequest                               = "100m"
+	DefaultMemoryRequest                            = "8Mi"
+	DefaultEmulatedMachines                         = "q35*,pc-q35*"
```

Comment:
Perhaps this should be generated from a `DefaultMachineType`? 
What happens if `DefaultMachineType` and `DefaultEmulatedMachines` differs?

Reply:
I don't think we can generate it from the `DefaultMachineType` - `DefaultEmulatedMachines` represent the machine **types** that can be used in the cluster while `DefaultMachineType` is the one the VMI is assigned with when no machine type is specified
While this is unrelated to this patch, I agree that we should prevent those two from being set with conflicting values (the default machine type was added after the supported types and it wasn't taken into account) - I've added a test for this.